lib/watir-webdriver/wait.rb

```diff
@@ -94,16 +94,9 @@ def wait_while(*args, &blk)
     end
   end
 
-  #
-  # Wraps an Element so that any subsequent method calls are
-  # put on hold until the element is present (exists and is visible) on the page.
-  #
-
-  class WhenPresentDecorator
+  class BaseDecorator
     extend Forwardable
```

Comment:
Just caught my eyes.

Since delegator is not used in base decorator, let's move this `extend` to `WhenPresentDecorator`.